src/ext/GeometryUtil.js

```diff
@@ -70,7 +70,7 @@
 
 			if (isMetric) {
 				units = ['ha', 'm'];
-				type = typeof isMetric;
+				const type = typeof isMetric;
```

Comment:
@szydan, window.type doesn't get assigned the `boolean` when this type variable is localised. The only place this type variable seems to be used is within this if statement and everything we need to use works on manual check. The other files in this PR (other than the version change) are relating to the build script.